src/shared/components/annotation/styles/civicCard.scss

```diff
@@ -0,0 +1,172 @@
+.civic-card {
```

Comment:
I removed the copied part from `oncoKbCard.scss`. Now this file only contain styles that are used by CIViC.